Web/Html,CSS,Javascript/verify-account-ui/index.html

```diff
@@ -11,12 +11,12 @@
       <h2>Verify Your Account</h2>
       <p>We emailed you the six digit code to cool_guy@email.com <br/> Enter the code below to confirm your email address.</p>
       <div class="code-container">
-        <input type="number" class="code" placeholder="0" min="0" max="9" required>
-        <input type="number" class="code" placeholder="0" min="0" max="9" required>
-        <input type="number" class="code" placeholder="0" min="0" max="9" required>
-        <input type="number" class="code" placeholder="0" min="0" max="9" required>
-        <input type="number" class="code" placeholder="0" min="0" max="9" required>
-        <input type="number" class="code" placeholder="0" min="0" max="9" required>
+        <input type="text" class="code" placeholder="0" maxlength="1" required>
```

Comment:
See this [comment](https://github.com/DevilsAutumn/Hacktoberfest22/pull/26#discussion_r986884434)